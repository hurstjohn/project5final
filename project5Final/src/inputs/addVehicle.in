2
1
123ABC
30
2
2
456DEF
1000.53
2
3
789GHI
200402.12
6
