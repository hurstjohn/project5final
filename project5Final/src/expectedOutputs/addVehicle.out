==========Options==========
1) Add Package
2) Add Vehicle
3) Activate Prime Day
4) Send Vehicle
5) Print Statistics
6) Exit
===========================

Vehicle Options:
1) Truck
2) Drone
3) Cargo Plane
Enter License Plate No.:
Enter Maximum Carry Weight:
==========Options==========
1) Add Package
2) Add Vehicle
3) Activate Prime Day
4) Send Vehicle
5) Print Statistics
6) Exit
===========================

Vehicle Options:
1) Truck
2) Drone
3) Cargo Plane
Enter License Plate No.:
Enter Maximum Carry Weight:
==========Options==========
1) Add Package
2) Add Vehicle
3) Activate Prime Day
4) Send Vehicle
5) Print Statistics
6) Exit
===========================

Vehicle Options:
1) Truck
2) Drone
3) Cargo Plane
Enter License Plate No.:
Enter Maximum Carry Weight:
==========Options==========
1) Add Package
2) Add Vehicle
3) Activate Prime Day
4) Send Vehicle
5) Print Statistics
6) Exit
===========================

